README.md

```diff
@@ -1,17 +1,160 @@
 
-See the node.js `local-dev-lib` library [README](https://github.com/HubSpot/hubspot-local-dev-lib)
+## Getting started
 
-For information on the `hubspot.config.yml` file, see [HubspotConfigFile](./docs/HubspotConfigFile.md)
+For more information on using these tools, see [Local Development Tooling: Getting Started](https://developers.hubspot.com/docs/cms/guides/getting-started-with-local-development)
 
-## Contributing
+### Installation
 
-For more information on developing, see the [Contributing Guide](CONTRIBUTING.md).
+```bash
+npm install -g @hubspot/cli
+```
+
+### Configuring
+
+Once the `@hubspot/cli` has been added to a project, a config file named [hubspot.config.yml](../../docs/HubspotConfigFile.md) will also be needed. It is recommended that the config file is kept in your `$HOME` directory.
+
+```bash
+cd ~
+hs init
+```
+
+#### Auto Completion
+
+You can set up command autocompletion by running
+
+```bash
+hs completion
+```
+
+and copying the output to either your `.bashrc` or `.zshrc`, and then sourcing that file `source ~/.bashrc` `source ~/.zshrc` or restarting your terminal.
+
+## Commands
+
+A full breakdown of the commands can be found on the [local development tools reference page](https://developers.hubspot.com/docs/cms/developer-reference/local-development-cli).
+
+**Note:** When `@hubspot/cli` is installed local to a project, the commands need to be prefixed with either `yarn` if using `yarn` or `npx` if using `npm`.
+
+Initialize the CLI and create a config file
+
+```bash
+hs init
+```
+
+Show all commands
+
+```bash
+hs help
+```
+
+Upload a file or directory to the Design Manager
+
+```bash
+hs upload --account=DEV [src] [dest]
+```
+
+Fetch a file or directory by path from the Design Manager
+
+```bash
+hs fetch --account=DEV [path] [dest]
+
+# Overwrite existing files
+hs fetch --account=DEV --overwrite [path] [dest]
+```
+
+Watch a directory of files and automatically upload changes to the Design Manager
+
+```bash
+hs watch --account=DEV [src] [dest]
+```
+
+Create a new asset locally
+
+```bash
+hs create [type] [dest]
+```
+
+Delete a file or directory from the Design Manager
+
+```bash
+hs remove --account=DEV [path]
+```
+
+Authenticate against an account using either `personalaccesskey` or `oauth2`
+
+```bash
+hs auth personalaccesskey
+```
+
+### File Manager Commands
+
+Upload a file or directory to the File Manager
+
+```bash
+hs filemanager upload --account=DEV [src] [dest]
+```
+
+Fetch a file or directory from the File Manager
+
+```bash
+hs filemanager fetch --account=DEV [src] [dest]
+
+# Overwrite existing files
+hs filemanager fetch --account=DEV --overwrite [path] [dest]
+```
+
+### HubDB Commands
+
+Create a new HubDB table
+
+```bash
+hs hubdb create <src>
+```
+
+Fetch a HubDB table
+
+```bash
+hs hubdb fetch <id or name> <src>
+```
+
+Clear all rows in a HubDB table
+
+```bash
+hs hubdb clear <id or name>
+```
+
+Delete a HubDB table
+
+```bash
+hs hubdb delete <id or name>
+```
```

Comment:
Do we want to take the opportunity to give a rundown of project and sandbox commands here?